Description: This sets the default svdrp-port to 0, which means that
 SVDRP is disabled by default unless someone specifies
 another port with the --port option when starting vdr
Author: Thomas Schmidt <tschmidt@debian.org>

Index: vdr/vdr.c
===================================================================
--- vdr.orig/vdr.c	2015-02-12 18:35:32.103236816 +0100
+++ vdr/vdr.c	2015-02-12 18:35:32.103236816 +0100
@@ -187,7 +187,7 @@
   // Command line options:
 
 #define dd(a, b) (*a ? a : b)
-#define DEFAULTSVDRPPORT 6419
+#define DEFAULTSVDRPPORT 0
 #define DEFAULTWATCHDOG     0 // seconds
 #define DEFAULTVIDEODIR VIDEODIR
 #define DEFAULTCONFDIR dd(CONFDIR, VideoDirectory)
